### DISCUSSION TITLE
Transfer domain from one project to another within the same account/team

### DISCUSSION THREAD

**Discussion by kristjanmar** [vercel/community#638] [Help] [answered]
Is there a recommended way to transfer a custom domain from one project to another project but within the same account or team?

As in, I have two projects deployed on Vercel. Project A is my production site using the custom domain, project B is a redesign of my site deployed through a different Github repository.

I want to switch my domain over from project A to project B, without downtime if possible.

I didn't find instructions on this in the docs, only about moving domains between individual accounts.

**Comment by amyegan** [comment, marked as answer]
Hi @kristjanmar,

We don’t currently have a streamlined way to transfer a custom domain between projects on the same account or team. The best way to accomplish this right now is to remove the custom domain from Project Settings (e.g. <span>http<span>s://vercel.co</span>m/$username/$projectname/settings/domains) of the project that is currently using the domain, then add it to the new project.
